src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIContainerService.java

```diff
@@ -406,18 +407,25 @@ private void scriptForJavaKotlin(ProgrammingExercise programmingExercise, String
             }
             else {
                 buildScript.append("""
-                        chmod +x gradlew
-                        ./gradlew clean structuralTests
-                        if [ $? -eq 0 ]; then
-                            ./gradlew behaviorTests
-                        fi
+                        mvn clean test
+                        """);
+            }
+            if (hasStaticCodeAnalysis) {
```

Comment:
```suggestion
           else if (hasStaticCodeAnalysis) {
```
I think sequential test runs and SCA are not supported together (add least for bamboo). You can't select both in the client anyways. But it's not blocking for me, as this script generation will be removed soon anyways